app/views/data_access/retrieval.html.erb

```diff
@@ -78,7 +78,22 @@
                  data-method="post"
                  data-confirm="Are you sure you want to remove this order?  This action cannot be undone."
                  rel="nofollow">Cancel</a>
-              <!-- /ko -->
+            <!-- /ko -->
+            <!-- ko if: dropped_granules -->
+              <p>The following granules are dropped from this order because the data access method <strong data-bind="text: method_name"></strong> is not supported:</p>
```

Comment:
Please change to:

    The following granules will not be processed because they do not support the <strong data-bind="text: method_name"></strong> access method:

Note that despite what line 79 says, we try not to use the word "order" anywhere user-visible.